sdk/master_board_sdk/CMakeLists.txt

```diff
@@ -73,10 +71,14 @@ ADD_LIBRARY(${PROJECT_NAME} SHARED
   ${MASTER_BOARD_SRC}
 )
 TARGET_LINK_LIBRARIES(${PROJECT_NAME} pthread)
+target_include_directories(
+  ${PROJECT_NAME} PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
+                         $<INSTALL_INTERFACE:include>)
 install(
   TARGETS ${PROJECT_NAME}
   EXPORT ${TARGETS_EXPORT_NAME}
   DESTINATION lib)
+install(DIRECTORY include/ DESTINATION include)
```

Comment:
the JRL-CMakeModules is supposed to automatically installs includes. If you remove this directive, do you lack any header in the install prefix ?

Reply:
Without this instruction the includes are not installed properly.
Only the `include/master/board/sdk/` `config.hh`|`deprecated.hh`|`warning.hh`

Reply:
Ok, that's because they are not included in `${${PROJECT_NAME}_HEADERS}`